src/main/java/com/diamondfire/helpbot/bot/command/impl/other/SamQuotesCommand.java

```diff
@@ -437,7 +480,35 @@ public void run(CommandEvent event) {
     public ArgumentSet compileArguments() {
         return new ArgumentSet()
                 .addArgument("action",
-                        new SingleArgumentContainer<>(new DefinedObjectArgument<>("submit", "generate", "get")).optional(null));
+                        new SingleArgumentContainer<>(new DefinedObjectArgument<>("submit", "generate", "get", "reload", "count")).optional(null));
+    }
+    
+    private void AddSamquote(String samquote) {
```

Comment:
Please follow the standard java naming convention. https://google.github.io/styleguide/javaguide.html

So this would be called ``addSamQuote``